tests/view_tests/tests/test_i18n.py

```diff
@@ -11,13 +11,14 @@
 from django.utils.translation import (
     LANGUAGE_SESSION_KEY, get_language, override,
 )
+from django.views.i18n import get_formats
 
 from ..urls import locale_dir
 
 
 @override_settings(ROOT_URLCONF='view_tests.urls')
-class I18NTests(TestCase):
-    """ Tests django views in django/views/i18n.py """
+class SetLanguageTests(TestCase):
+    """Test the django.views.i18n.set_language view."""
```

Comment:
I guess this could be done in a separate commit from adding the test as it's not really related. It seems there's more cleanup to do as "Tests views in django/views/i18n.py that need to change settings.LANGUAGE_CODE." for JsI18NTests seems inaccurate.